src/camel_snake_kebab/internals/macros.cljc

```diff
@@ -1,7 +1,12 @@
 (ns ^:no-doc camel-snake-kebab.internals.macros
+  #?(:cljs (:refer-clojure :exclude [resolve]))
   (:require [camel-snake-kebab.internals.alter-name :refer [alter-name]]
             [camel-snake-kebab.internals.misc :refer [convert-case]]))
 
+#?(:cljs
```

Comment:
Add an inline rationale comment on why this kind of sourcery is required and what it achieves.

Reply:
Done